src/main/kotlin/no/nav/familie/ef/sak/opplysninger/personopplysninger/fullmakt/FullmaktClient.kt

```diff
@@ -0,0 +1,45 @@
+package no.nav.familie.ef.sak.opplysninger.personopplysninger.fullmakt
+
+import no.nav.familie.http.client.AbstractPingableRestClient
+import org.apache.hc.client5.http.utils.Base64
+import org.springframework.beans.factory.annotation.Qualifier
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.stereotype.Service
+import org.springframework.web.client.RestOperations
+import java.net.URI
+import java.time.LocalDate
+
+@Service
+class FullmaktClient(
+    @Value("\${PDL_FULLMAKT_URL}")
+    private val fullmaktUrl: String,
+    @Qualifier("azure")
+    private val restOperations: RestOperations,
+) : AbstractPingableRestClient(restOperations, "repr.fullmakt") {
+    override val pingUri: URI = URI.create(fullmaktUrl)
+
+    fun hentFullmakt(ident: String): List<FullmaktResponse> {
+        val url = URI.create("$fullmaktUrl/api/internbruker/fullmektig")
+        val base64EncodedIdent = Base64.encodeBase64String(ident.toByteArray())
```

Comment:
er det på grunn av logging at de har gjort dette slik?

Reply:
Tror ikke det er noen god grunn, dessverre. Base64 encoding skal ikke regnes som noen form for kryptering og at logging av fnr i base64 ville i praksis vært det samme som å logge fnr i klartekst.